### PR TITLE
UpdatingCron Configuration Document

### PR DESCRIPTION
**Pull request by manavluhar**
## Purpose of this pull request
This pull request (PR) for updating Cron Configuration documentation in Configuration Guide by resolving styling and wrong output issues.

- For **1**st Page listed underneath, I have updated the warning callout pattern by standard one to resolve styling-related issues(*bullet pointers were touching the outline of warning callout section*).
- For **2**st Page listed underneath, I have updated the info callout pattern by standard one to resolve styling-related issues(*bullet pointers were touching the outline of info callout section*). Along with this updated command by adding `bin` directory path before the command & updated the output of `crontab -l`  by generalizing the actual result of the same.

## Affected DevDocs pages
1 : https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-cron.html#create-or-remove-the-magento-crontab : setup-cron_about.md

2 : https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-cron.html#create-the-commerce-crontab : setup-cron_how-to.md

### CODE REVIEW

File: src/_includes/config/setup-cron_how-to.md [hunk 1]
```diff
@@ -31,7 +37,7 @@ A sample follows:
 
 ```terminal
 #~ MAGENTO START c5f9e5ed71cceaabc4d4fd9b3e827a2b
-* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run 2>&1 | grep -v "Ran jobs by schedule" >> /var/www/html/magento2/var/log/magento.cron.log
+* * * * * /usr/bin/php<php_version> <install_directory>/bin/magento cron:run 2>&1 | grep -v "Ran jobs by schedule" >> <install_directory>/var/log/magento.cron.log
```
Comment (hguthrie) [inline src/_includes/config/setup-cron_how-to.md:40]: Although generalizing would be nice, the full path seems to be used heavily throughout the cron topics. You'd have to address the consistency, so maybe this change is out of scope.

File: src/_includes/config/setup-cron_about.md [hunk 2]
```diff
@@ -4,7 +4,13 @@ The Magento _crontab_ is the configuration used to run Magento cron jobs.
...
+{%
+include note.html
+type='warning'
+content='
 
 *  To avoid issues during installation and upgrade, we strongly recommend you apply the same PHP settings to both the PHP command-line configuration and to the PHP web server plug-in's configuration. For more information, see [Required PHP settings]({{ page.baseurl }}/install-gde/prereq/php-settings.html).
+
 *  In a multi-node system, crontab can run on only one node. This applies to you only if you set up more than one webnode for reasons related to performance or scalability.
+
+'%}
```
Comment (dshevtsov) [inline src/_includes/config/setup-cron_about.md:16]:
```suggestion
<div class="bs-callout-warning" markdown="1">

*  To avoid issues during installation and upgrade, we strongly recommend you apply the same PHP settings to both the PHP command-line configuration and to the PHP web server plug-in's configuration. For more information, see [Required PHP settings]({{ page.baseurl }}/install-gde/prereq/php-settings.html).
*  In a multi-node system, crontab can run on only one node. This applies to you only if you set up more than one webnode for reasons related to performance or scalability.

</div>
```

Liquid tags cannot contain Liquid syntax, in this case, `{{ page.baseurl }}`

File: src/_includes/config/setup-cron_how-to.md [hunk 3]
```diff
@@ -16,10 +16,16 @@ To create the Commerce crontab:
...
 Use `--force` to rewrite an existing Magento crontab.
 
- {:.bs-callout-info}
+{%
+include note.html
+type='info'
+content='
+
+*  `bin/magento cron:install` does not rewrite an existing crontab inside `#~ MAGENTO START` and `#~ MAGENTO END` comments in your crontab.
+
```
Comment (dshevtsov) [inline src/_includes/config/setup-cron_how-to.md:25]:
```suggestion
```

No need to add an empty line here.

File: src/_includes/config/setup-cron_how-to.md (hunk 1 above)
Comment (dshevtsov) [reply src/_includes/config/setup-cron_how-to.md:40]: Typically, we use generalization to describe a format of a command or something else. When we are talking about a sample, we provide a particular case without generalization.

**Review by dshevtsov — CHANGES_REQUESTED** [review]

**Comment by hguthrie** [comment]
It looks like these things were addressed during the migration:

- https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/cli/configure-cron-jobs.html#create-or-remove-the-commerce-crontab
- https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/cli/configure-cron-jobs.html#create-the-commerce-crontab

Closing this PR. If you still see issues with this content, you can report an issue or create a PR in the new repository.
